src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -1081,7 +1081,7 @@ def call(
 
         decoder_outputs = self.decoder(
             inputs["decoder_input_ids"],
-            attention_mask=decoder_attention_mask,
+            attention_mask=inputs["decoder_attention_mask"],
```

Comment:
Missed that in the previous TF Bart PR. Should not have been not too much of a problem as it only breaks in an edge-case.